crowbar_framework/app/models/proposal_object.rb

```diff
@@ -126,14 +126,14 @@ def save(options = {})
     increment_crowbar_revision!
     Rails.logger.debug("Saving data bag item: #{@item["id"]} - #{crowbar_revision}")
     @item.save
-    save_proposal_in_sqlite
+    save_proposal_in_sqlite unless options.fetch(:skip_sync, false)
```

Comment:
is there any case where we would have `:skip_sync, false`? it does not hurt but I don't see the necessity here if it's true in all cases.
and maybe my notion of the sync is wrong but shouldn't it just be called `sync`? with `skip_sync, true` it should not sync